packages/nextjs/.gitignore

```
@@ -0,0 +1 @@
+scripts/build/
```

nit

```suggestion
/scripts/build/
```

`scripts/build/` would match at any depth, for example, `/packages/nextjs/a/future/path/scripts/build/` would be ignored.
`/scripts/build/` only matches one specific path.